process/transaction/metaProcess.go

```diff
@@ -138,17 +138,26 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) (
 }
 
 // VerifyTransaction verifies the account states in respect with the transaction data
-func (txProc *metaTxProcessor) VerifyTransaction(tx *transaction.Transaction) (state.UserAccountHandler, error) {
+func (txProc *metaTxProcessor) VerifyTransaction(tx *transaction.Transaction) error {
```

Comment:
no need to overload the VerifyTransaction and GetSenderAndReceiverAccounts in metaProcess.go. They are defined in the baseProcess.go and look identical.

Reply:
Removed